.github/workflows/release.yml

```diff
@@ -2,24 +2,37 @@ on:
   push:
     # Sequence of patterns matched against refs/tags
     tags:
-      - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
 
-name: Create Release
+name: Release
 
 jobs:
   build:
-    name: Create Release
+    name: Create Release and Upload Asset
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
+      - name: Build project
+        run: |
+          zip indigemojis-${{ github.ref }} dist/*
```

Comment:
I am just running zip here but this could also do the build phase for use as well. I think that would be something like `indigemojis all` or something like that.